modules/client-configuration/pages/clients-debian.adoc

```diff
@@ -68,7 +68,7 @@ The channels you need for this procedure are:
 
 |===
 
-include::snippets/add_channels_cli.adoc[]
+include::snippets/add_channels_ssc.adoc[]
```

Comment:
No. No SCC for now for Debian.

Reply:
You listed spacewalk-common-channels for Debian under both Uyuni and SUMA:

![image](https://user-images.githubusercontent.com/3914967/98321294-7944c080-2030-11eb-978f-d9c4b6ee8e08.png)

Reply:
Same story.